gui/js/onboarding.window.js

```diff
@@ -113,19 +121,29 @@ module.exports = class OnboardingWM extends WindowManager {
   }
 
   onChooseFolder (event) {
+    // FIXME: The modal may appear on background, either every time (e.g. Ubuntu)
+    // or only the second time (e.g. Fedora)
     let folders = dialog.showOpenDialog({
       properties: ['openDirectory', 'createDirectory']
     })
     if (folders && folders.length > 0) {
-      const result = this.desktop.checkSyncPath(folders[0])
-      event.sender.send('folder-chosen', {
-        folder: result.syncPath,
-        error: result.error ? `Folder ${result.error}` : null
-      })
+      this.checkSyncPath(folders[0], event.sender)
     }
   }
 
+  checkSyncPath (syncPath, eventSender) {
+    console.log('checkSyncPath', syncPath, eventSender.send)
+    const result = this.desktop.checkSyncPath(syncPath)
+    eventSender.send('folder-chosen', {
+      folder: result.syncPath,
+      error: result.error ? `Folder ${result.error}` : null
+    })
+    return result
+  }
+
   onStartSync (event, syncPath) {
+    const {error} = this.checkSyncPath(syncPath, event.sender)
+    if (error) return
```

Comment:
`log.warn` it,  just in case ?

Reply:
Makes sense. What about doing so directly in `checkSyncPath`?

Reply:
It will log for everyone who attempt to sync an existing folder (before they are given a warning it's not possible), whereas doing it here will only log if the user manage to ignore the error and click "start sync" with a bad folder?.

Reply:
:+1: